src/V3PerPackage/V3PerPackage.csproj

```diff
@@ -135,6 +140,9 @@
     <SignPath>..\..\build</SignPath>
     <SignPath Condition="'$(BUILD_SOURCESDIRECTORY)' != ''">$(BUILD_SOURCESDIRECTORY)\build</SignPath>
     <SignPath Condition="'$(NuGetBuildPath)' != ''">$(NuGetBuildPath)</SignPath>
+    <SignType Condition="'$(SignType)' == ''">none</SignType>
   </PropertyGroup>
   <Import Project="$(SignPath)\sign.targets" Condition="Exists('$(SignPath)\sign.targets')" />
+  <Import Project="$(SignPath)\sign.microbuild.targets" Condition="Exists('$(SignPath)\sign.microbuild.targets')" />
```

Comment:
AFAIK, V3PerPackage doesn't need signing. It's a tool run locally.

Reply:
I think we build the `.nupkg`, we just don't upload it to Octopus.
Because this project is building an actual `.exe` I'd much rather we sign it now rather than find out later it's not signed.

Reply:
Okay, sounds good.